docs/modules/ROOT/partials/cmos-health-checks.adoc

```diff
@@ -158,6 +158,15 @@ Dropped ticks are usually a sign of resource contention, specifically CPU conten
 
 *Further Reading*: xref:7.0@server:install:sizing-general[Sizing Guidelines]
 
+[#CB90067]
+=== Service Has Been Forced to Roll Back to Zero (CB90067)
+
+*Background*: A service has been forced to roll back to zero. This is bad because it means the service is attempting to resynchronise the entirety of the data set.
+
+*Condition*: a rollback is seen in the `memcached` logs.
+
+*Remediation*: Contact Couchbase Technical Support.
```

Comment:
Is there anything more immediately actionable that we can have here?

Reply:
in the linked cbse and by extension the linked ticket, I can't see what the solution was to this issue. I figured it was more of a case by case problem. If I'm wrong please do tell me and I'll put a better remediation in there.

Reply:
The issue is that this is a very generic problem - a rollback to zero is a symptom, but there are many possible causes for it. I'll have a think if we can have anything more specific here.

If I can't think of anything, it may be worth saying something like "this might not be a problem, but keep an eye on your cluster and contact support if you notice any problems" (obviously worded more professionally than that)